mix.exs

```diff
@@ -54,12 +54,12 @@ defmodule Xav.MixProject do
 
   defp deps do
     [
-      {:nx, "~> 0.7.0", optional: true},
+      {:nx, "~> 0.9.0", optional: true},
```

Comment:
This library only uses `Nx.reshape` and `Nx.from_binary`, so it should be totally fine to relax this requirement, we can do `~> 0.7` (it could totally be earlier also) :)